osmaxx/conversion/converters/converter_gis/helper/osm_importer.py

```diff
@@ -0,0 +1,94 @@
+from itertools import chain
+from sqlalchemy import MetaData, Table, create_engine, func
+from sqlalchemy.engine.url import URL
+from sqlalchemy.sql import select, insert
+# import needed for reflection to work correctly with spatial types
+from geoalchemy2 import Geometry, Geography  # noqa
+
+
+class OSMImporter:
+    def __init__(self):
+        self._osm_derived_tables = ['osm_point', 'osm_line', 'osm_polygon', 'osm_roads']
+        self._osm_base_tables = ['osm_ways', 'osm_nodes']
+
+        _world_db_connection_parameters = {
+            'username': 'gis',
+            'password': 'gis',
+            'port': 5432,
+            'database': 'gis',
+            'host': 'world-database',
+        }
+        world_db_connection = URL('postgresql', **_world_db_connection_parameters)
+        self._world_db_engine = create_engine(world_db_connection, use_native_hstore=True)
+        self._world_db_meta_data = MetaData()
+
+        _local_db_connection_parameters = {
+            'username': 'postgres',
+            'password': 'postgres',
+            'port': 5432,
+            'database': 'osmaxx_db',
+        }
+        local_db_connection = URL('postgresql', **_local_db_connection_parameters)
+        self._local_db_engine = create_engine(local_db_connection, use_native_hstore=True)
+        self._table_metas = {}
+
+        for to_be_created_table in self._osm_derived_tables + self._osm_base_tables:
```

Comment:
Consider using a dictionary comprehension to combine assignment and content generation.